app/models/maintenance_tasks/progress.rb

```diff
@@ -51,7 +53,9 @@ def text
       count = @run.tick_count
```

Comment:
Perhaps we could do this instead?
```suggestion
      count = @run.tick_count || 0
```

And leave the if statement intact?